lago/plugins/vm.py

```diff
@@ -360,7 +360,7 @@ def wait_for_ssh(self):
         return ssh.wait_for_ssh(
             ip_addr=self.ip(),
             host_name=self.name(),
-            connect_retries=self._spec.get('boot_time_sec', 50),
+            connect_timeout=self._spec.get('boot_time_sec', 600),
```

Comment:
To tweak the ssh timeout you have the ssh_timeout conf value:

https://github.com/lago-project/lago/pull/287/files#diff-031917605b8ea7e9f49d56da61e00f6eR319

Not sure if that's what you wanted here.